lib/jekyll/tags/highlight.rb

```diff
@@ -66,16 +53,38 @@ def sanitized_opts(opts, is_safe)
             [:startinline, opts.fetch(:startinline, nil)],
             [:hl_lines,    opts.fetch(:hl_lines, nil)],
             [:linenos,     opts.fetch(:linenos, nil)],
-            [:encoding,    opts.fetch(:encoding, 'utf-8')],
+            [:encoding,    opts.fetch(:encoding, "utf-8")],
             [:cssclass,    opts.fetch(:cssclass, nil)]
           ].reject { |f| f.last.nil? }]
         else
           opts
         end
       end
 
+      private
+
+      def parse_options(input)
+        options = {}
+        if defined?(input) && input != ""
```

Comment:
Do we need that check at all? Maybe it would be enough to do it like this:
`
unless input == ""
`